static/app/components/events/attachmentViewers/jsonViewer.tsx

```diff
@@ -1,53 +1,56 @@
 import styled from '@emotion/styled';
 
 import ContextData from 'sentry/components/contextData';
-import DeprecatedAsyncComponent from 'sentry/components/deprecatedAsyncComponent';
 import PreviewPanelItem from 'sentry/components/events/attachmentViewers/previewPanelItem';
 import {
   getAttachmentUrl,
   ViewerProps,
 } from 'sentry/components/events/attachmentViewers/utils';
+import LoadingError from 'sentry/components/loadingError';
+import LoadingIndicator from 'sentry/components/loadingIndicator';
+import {t} from 'sentry/locale';
+import {useApiQuery} from 'sentry/utils/queryClient';
+
+export default function NewJsonViewer(props: ViewerProps) {
+  const query = useApiQuery([getAttachmentUrl(props)], {
```

Comment:
i think headers can be passed in
```suggestion
  const query = useApiQuery([getAttachmentUrl(props), {headers: xyz}], {
```